createuserpkg

```diff
@@ -50,12 +53,17 @@ def main():
     options, arguments = parser.parse_args()
 
     # verify options and arguments
-    required_options = ('name', 'uid', 'password', 'version', 'identifier')
+    required_options = ('name', 'uid', 'version', 'identifier')
     missing_required_options = False
     for option in required_options:
         if not hasattr(options, option) or getattr(options, option) == None:
             print >> sys.stderr, 'Missing required option: %s' % option
             missing_required_options = True
+    haspassword = getattr(options, 'password') is not None
+    hasprompt = options.prompt == True
+    if haspassword == hasprompt:
+        print >> sys.stderr, 'Must provide password or prompt option'
+        missing_required_options = True
```

Comment:
I wonder, though, if it might just be more friendly to prompt if -p isn't given...

Reply:
IOW, use the password passed in via -p, else prompt. We could then either leave -P in or not, it really wouldn't matter.

Reply:
Yep, that works too, your call.

Reply:
Removing -P for this would free that up for "picture", should someone want to add that...

Reply:
Done in 544c5d0